video_xblock/static/js/studio-view/studio-edit-transcripts-autoload.js

```diff
@@ -10,29 +10,14 @@ function createStatusMessageElement(langCode, actionSelector) {
 
     var parentSelector = '';
     var messageSelector = '.api-response.' + actionSelector + '.' + langCode + '.status';
-    var $messageUpload;
     if (actionSelector === 'upload-default-transcript') {
         parentSelector = 'available-default-transcripts-section';
     } else if (actionSelector === 'remove-default-transcript') {
         parentSelector = 'enabled-default-transcripts-section';
     }
 
     if ($(messageSelector).length === 0) {
-        $messageUpload = $('<div>',
-            {class: messageSelector});
-        $messageUpload.appendTo($('.' + parentSelector + ':visible').last());
-    }
-}
-
-/**
- *  Manage default transcripts labels display depending on enabled/available subs presence.
- */
-function setDisplayDefaultTranscriptsLabel(isNotDisplayedDefaultSub, labelElement) {
-    'use strict';
-    if (isNotDisplayedDefaultSub) {
-        labelElement.addClass('is-hidden');
-    } else {
-        labelElement.removeClass('is-hidden');
+        $('<div>', {class: messageSelector}).appendTo($('.' + parentSelector + ':visible').last());
```

Comment:
quite weird line of code. Can we use some class name instead of `$('.' + parentSelector + ':visible'`. What is it ?